lib/types/api.types.ts

```diff
@@ -78,10 +78,14 @@ export interface LocationAPI {
 
 /* Parameters API */
 
-export interface ParametersAPI {
-  installation: KeyValueMap
-  instance: KeyValueMap
-  invocation?: SerializedJSONValue
+export interface ParametersAPI<
+  TInstallationParameters extends KeyValueMap,
+  TInstanceParameters extends KeyValueMap,
+  TInvocationParameters extends SerializedJSONValue
+> {
+  installation: TInstallationParameters
+  instance: TInstanceParameters
+  invocation?: TInvocationParameters
```

Comment:
This should not be optional. We are passing `never` when it's no available

Reply:
good catch – fixed!